Source/GoBot/ViewDatabase.swift

```diff
@@ -443,6 +443,23 @@ class ViewDatabase {
         return -1
     }
     
+    func lastWrittenMessageDate() throws -> Date? {
```

Comment:
The function name is descriptive, but a comment explaining the intended use (useful for knowing the last time the user synced) may help developers understand better what it is used for

Reply:
good call